features/yield/Yield.tsx

```diff
@@ -124,6 +124,18 @@ const Yield = () => {
               {exchangeName} Exchange Link
             </Link>{" "}
             ← Go here to check the {tokenSymbol} exchange rate
+            <br></br>
+            <br></br>
+            <Typography>
+              <strong>APY</strong>: To approximate your weekly returns, you can
+              divide the number of UMA rewarded for a pool per week by the $
+              amount of pooled liquidity. For example, if the pool size is $10
+              million and the reward is 25,000 UMA/week, then the return for
+              each $1 supplied to the pool is:{" "}
+              <strong>
+                (25,000 / $10 million) = 0.0025 UMA per week per $1
```

Comment:
Do you want to also add the UMA price in there to make it a typical APY percentage? Something like:
```
($10 * 25,000 / $10 million) * 100 = 2.5% per week
```

Reply:
great suggestion